### PR TITLE
fix(views): only output subtitle element in object summary if provided

### LINKED ISSUE #9759: object/elements/summary should not output subtext if empty subtitle

### PR DESCRIPTION
**Pull request by jdalsem**
fixes #9759

### CODE REVIEW

File: views/default/object/elements/summary.php [hunk 1]
```diff
@@ -49,7 +49,9 @@
 if ($title_link) {
 	echo "<h3>$title_link</h3>";
 }
-echo "<div class=\"elgg-subtext\">$subtitle</div>";
+if ($subtitle) {
```
Comment (hypeJunction) [inline views/default/object/elements/summary.php]: Maybe should still output an empty string if set to ''?

**Comment by hypeJunction** [comment]
Commit message type should be `fix`

File: views/default/object/elements/summary.php (hunk 1 above)
Comment (mrclay) [inline views/default/object/elements/summary.php]: User can use ` ` (one blank space) or `<!-- -->` to get an empty element if that's what you mean.

**Comment by mrclay** [comment]
While our BC policy doesn't guarantee markup in views it would be nice to add a bit about the effect of this in guides/upgrading.rst. Someone's layout CSS could depend on that element always being present.

**Comment by mrclay** [comment]
`fix` in the summary, `Fixes` in the description. I'd be in favor of altering our system to accept this commit as is.

**Comment by jdalsem** [comment]
I have updated the commit message. I feel it is overkill to add info about this change in the upgrade documentation

**Comment by mrclay** [comment]
I think 2.x is the place for this change, but I won't block it. We should be reserving 1.12 and 2.1 for clear cut bugs and this is just an opinion about empty markup being present.

**Comment by hypeJunction** [comment]
Do you still want this in 1.12, or is #9944 enough?

**Comment by jdalsem** [comment]
#9944 is fine by me

**Pull request closed without merging**